importer/templates/importer/upload.html

```diff
@@ -0,0 +1,6 @@
+<h1>upload here</h1>
```

Comment:
No arguments there. There's no base.html yet and there's no HTML header or anything in these templates that would conflict with it once it exists. I'll create an empty base.html and extend it just so it's there.